app/models/katello/host/subscription_facet.rb

```diff
@@ -34,6 +35,19 @@ def import_database_attributes(consumer_params)
         end
       end
 
+      def update_consumer_type(consumer_params)
+        if consumer_params.try(:[], 'type').try(:[], 'label') == 'hypervisor'
+          self.consumer_type = 'hypervisor'
+        elsif consumer_params.try(:[], 'facts').try(:[], 'virt.is_guest') == "true"
```

Comment:
If physical and virtual are important, and distinct from hypervisor and guest then lets agree to that and let me build that into this design. 

If the above is true, what would we call physical vs virtual vs hypervisor vs guest? I don't feel like a single field would encapsulate all those concerns. We could do the following fields (granted this adds more columns):

```
physical (boolean)
virtual (boolean)
consumer_type (string) values: hypervisor or guest
```

The `consumer_type` is very clearly a subscription facet, however, the physical and virtual get into the realm of the host properties directly. Today these values could be searched by facts with pre-made bookmarks to help users however that is not as obvious if we consider these attributes to be first class citizens of what a host is.